Adjust the maturity warning for ECS to be aligned with GA / SemVer

webmat opened the pull request:
This PR contains a few changes:

- Adjusting the wording of the maturity warning in the Markdown and Asciidoc files
- Removal of the asciidoc beta label
- Replace the warning mention at the top of the readme to be less maintenance to maintain

MikePaquette commented on docs/index.asciidoc:
```diff
@@ -14,20 +14,27 @@ ingesting data into Elasticsearch. A common schema helps you correlate
+[[ecs-maturity]]
+=== Maturity
+
+With ECS turning 1.0, the team will will approach improvements by following SemVer.
```
As ECS turns 1.0, further development will follow Semantic Versioning ([SemVer](https://semver.org/)).

MikePaquette commented on docs/index.asciidoc:
```diff
@@ -14,20 +14,27 @@ ingesting data into Elasticsearch. A common schema helps you correlate
+=== Maturity
+
+With ECS turning 1.0, the team will will approach improvements by following SemVer.
+This means we will only do breaking changes at major versions, and that
```
This means that breaking changes will be limited to major versions, and that

MikePaquette commented on docs/index.asciidoc:
```diff
@@ -14,20 +14,27 @@ ingesting data into Elasticsearch. A common schema helps you correlate
+
+With ECS turning 1.0, the team will will approach improvements by following SemVer.
+This means we will only do breaking changes at major versions, and that
+we will improve ECS in a backwards compatible manner in the meantime.
```
improvements will be implemented in a backwards-compatible manner in the meantime.

MikePaquette commented on docs/index.asciidoc:
```diff
@@ -14,20 +14,27 @@ ingesting data into Elasticsearch. A common schema helps you correlate
+With ECS turning 1.0, the team will will approach improvements by following SemVer.
+This means we will only do breaking changes at major versions, and that
+we will improve ECS in a backwards compatible manner in the meantime.
+We plan to align major ECS releases with major Elastic Stack releases.
```
Generally major ECS releases are planned to be aligned with major Elastic Stack releases.

MikePaquette commented on docs/intro.md:
```diff
@@ -20,9 +20,15 @@ ingesting data into Elasticsearch. A common schema helps you correlate
-For contributions please read the [Contributing Guide](CONTRIBUTING.md).
+## Maturity
+
+With ECS turning 1.0, the team will will approach improvements by following SemVer.
```
As ECS turns 1.0, further development will follow Semantic Versioning ([SemVer](https://semver.org/)).

MikePaquette commented on docs/intro.md:
```diff
@@ -20,9 +20,15 @@ ingesting data into Elasticsearch. A common schema helps you correlate
+## Maturity
+
+With ECS turning 1.0, the team will will approach improvements by following SemVer.
+This means we will only do breaking changes at major versions, and that
```
This means that breaking changes will be limited to major versions, and that

webmat replied:
Deja vu 😆

MikePaquette commented on docs/intro.md:
```diff
@@ -20,9 +20,15 @@ ingesting data into Elasticsearch. A common schema helps you correlate
+
+With ECS turning 1.0, the team will will approach improvements by following SemVer.
+This means we will only do breaking changes at major versions, and that
+we will improve ECS in a backwards compatible manner in the meantime.
```
improvements will be implemented in a backwards-compatible manner in the meantime.

MikePaquette replied:
Generally major ECS releases are planned to be aligned with major Elastic Stack releases.

MikePaquette commented on docs/intro.md:
```diff
@@ -20,9 +20,15 @@ ingesting data into Elasticsearch. A common schema helps you correlate
+With ECS turning 1.0, the team will will approach improvements by following SemVer.
+This means we will only do breaking changes at major versions, and that
+we will improve ECS in a backwards compatible manner in the meantime.
+We plan to align major ECS releases with major Elastic Stack releases.
```
Generally major ECS releases are planned to be aligned with major Elastic Stack releases.

webmat commented:
@MikePaquette I've addressed all of your points

I'm adding two more things as well:

- Removal of the asciidoc beta label
- Replace the warning mention at the top of the readme to be less maintenance to maintain

ruflin commented on README.md:
```diff
@@ -20,9 +20,15 @@ ingesting data into Elasticsearch. A common schema helps you correlate
-For contributions please read the [Contributing Guide](CONTRIBUTING.md).
+## Maturity
+
+With ECS turning 1.0, the team will approach improvements by following SemVer.
```
Instead of describing SemVer I would just link to SemVer: https://semver.org/

webmat replied:
Yeah I'm good with adding the link and stripping out the rest for now. If people constantly ask, I can add the short explainer again.

ruflin commented on README.md:
```diff
@@ -20,9 +20,15 @@ ingesting data into Elasticsearch. A common schema helps you correlate
+With ECS turning 1.0, the team will approach improvements by following SemVer.
+This means that breaking changes will be limited to major versions, and that
+improvements will be implemented in a backwards-compatible manner in the meantime.
+Generally major ECS releases are planned to be aligned with major Elastic Stack releases.
```
Is that the case? :-)
